src/EditorFeatures/Core/InlineRename/InlineRenameSessionOptionsStorage.cs

```diff
@@ -13,5 +14,13 @@ internal static class InlineRenameSessionOptionsStorage
     public static readonly Option2<bool> RenameInComments = new("dotnet_rename_in_comments", defaultValue: false);
     public static readonly Option2<bool> RenameFile = new("dotnet_rename_file", defaultValue: true);
     public static readonly Option2<bool> PreviewChanges = new("dotnet_preview_inline_rename_changes", defaultValue: false);
+
+    [Obsolete]
```

Comment:
This option is introduced and shown in the current UI option page, but since all the `CommitAsync` call is blocked by `JTF.Run`, it doesn't have any real effect.
Its default value is set to `true` but I want to set up an experiment for this, so I just simply mark it as `Obsolete`

Reply:
i'm confused.  you changed the UI option to use the two flags below.  so just remove this flag.